tests/src/main/middlewares/auth.test.ts

```diff
@@ -30,7 +31,7 @@ describe('Authentication Middleware', () => {
   })
 
   it('Should return 401 if token payload is empty', async () => {
-    const tokenWithPayloadEmpty = sign({ payload: {} }, 'secret')
+    const tokenWithPayloadEmpty = sign({ payload: {} }, env.secret)
```

Comment:
I think that is better to use the JwtAdapter otherwise a JWT pure implementation. What you think?

Reply:
This "sign" method is from jwt. @SamuelS00 imported just it instead of the whole module

Reply:
I think it's better to change this, by chance we change the JWT lib we will have to track where it is in the code.